arelle/RuntimeOptions.py

```diff
@@ -190,3 +190,10 @@ def __post_init__(self, pluginOptions: Optional[dict[str, RuntimeOptionValue]])
                 self.roleTypesFile, self.arcroleTypesFile
         )):
             raise RuntimeOptionsException(_('Incorrect arguments with webserver'))
+
+    @staticmethod
+    def assertOptionsEqual(name: str, optionsValue: Any, value: Any) -> Any:
```

Comment:
Agreed on the function name.

As far as the static implementation, doing something like this:
```
def requireNonConflictingOption(self, name: str, value: Any) -> Any:
        currentValue = getattr(self, name, None)
```
and
```
options.requireNonConflictingOption('logFile', logFileName)
```
means this code will not appear when someone is searching for references to `RuntimeOptions.logFile`. I wanted the implementation to force a direct reference to the property rather than using something like `getattr`, which led me to the implementation that had no use of `self` and thus a static method was appropriate. If there's some kind of trick I'm missing to get around this, or if that isn't a principle we care about, I'm fine with making the above change.